yarn.lock

```diff
@@ -5265,6 +5194,22 @@ cozy-client@^29.1.1:
     sift "^6.0.0"
     url-search-params-polyfill "^8.0.0"
 
+cozy-client@^6.58.0:
+  version "6.64.5"
+  resolved "https://registry.yarnpkg.com/cozy-client/-/cozy-client-6.64.5.tgz#33302c3190387ad42928189ae916f5afb28c3a84"
+  integrity sha512-DnlNaYNTeTAuIAGeKggQbGW/a9CUl19Ec9IdUzBlzkeylAkBvdfbyIg6BTzXePdae0o+bGh0LV9VAJ5yX4C2YA==
+  dependencies:
+    cozy-device-helper "^1.7.3"
+    cozy-stack-client "^6.64.3"
+    lodash "^4.17.13"
+    microee "^0.0.6"
+    prop-types "^15.6.2"
+    react-redux "^5.0.7"
+    redux "^3.7.2"
+    redux-thunk "^2.3.0"
+    sift "^6.0.0"
+    url-search-params-polyfill "^7.0.0"
```

Comment:
nit: So curious about this version. Can you make a `yarn why cozy-client` and put the result here? In order to see what needs to be updated?

Reply:
```
=> Found "cozy-client@29.1.1"
info Has been hoisted to "cozy-client"
info Reasons this module exists
   - Specified in "dependencies"
   - Hoisted from "cozy-pouch-link#cozy-client"
info Disk size without dependencies: "8.34MB"
info Disk size with unique dependencies: "19.93MB"
info Disk size with transitive dependencies: "27.67MB"
info Number of shared dependencies: 65
=> Found "cozy-konnector-libs#cozy-client@23.22.0"
info This module exists because "cozy-konnector-libs" depends on it.
info Disk size without dependencies: "1.75MB"
info Disk size with unique dependencies: "13.09MB"
info Disk size with transitive dependencies: "16.65MB"
info Number of shared dependencies: 52
=> Found "cozy-jobs-cli#cozy-client@6.64.5"
info Reasons this module exists
   - "cozy-jobs-cli#cozy-konnector-libs" depends on it
   - Hoisted from "cozy-jobs-cli#cozy-konnector-libs#cozy-client"
info Disk size without dependencies: "404KB"
info Disk size with unique dependencies: "7.34MB"
info Disk size with transitive dependencies: "8.74MB"
info Number of shared dependencies: 21
```